src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIConnectorService.java

```diff
@@ -243,6 +243,7 @@ private void processNewPushToRepository(ProgrammingExerciseParticipation partici
         programmingMessagingService.notifyUserAboutSubmission(submission, participation.getExercise().getId());
 
         // Trigger the build for the new submission on the local CI system.
+        // TODO: this is already invoked in the service method processNewProgrammingSubmission above, however without the commit hash, we should probably unify the methods
```

Comment:
Should this TODO be resolved with this PR? If not a Issue to track this would be nice.

Reply:
Created by Stephan. When I asked him about this in another PR he said he doesn't always want to create new issues.